emails/lib/spree/emails/engine.rb

```diff
@@ -12,6 +12,10 @@ def self.activate
         end
       end
 
+      initializer 'spree_emails.assets.precompile' do |app|
```

Comment:
@aleksandarpetrushev this is declared here: https://github.com/spree/spree/blob/main/emails/config/initializers/assets.rb